system-tests/fixtures/ios-container/package.json

```diff
@@ -1,8 +1,7 @@
 {
   "dependencies": {
-    "@react-native-community/cli-platform-ios": "4.8.0",
```

Comment:
Ah, so this is no longer required? Good, since I was getting some errors locally when they updated their package to `4.9.0`.